File: Python/PrimeX.py
NL = 1000000000

print("========\n PrimeX\n========\n")

f = open("primes.txt", "w")

N = NL + 2
primes = [True]*(N)

for i in range(2, 3):
    if primes[i]:
        f.write(str(i) + "\n")
        for j in range(i+i, N, i):
            primes[j] = False

for i in range(3, N, 2):
    if primes[i]:
        f.write(str(i) + "\n")
        for j in range(i+i, N, i):
            primes[j] = False

P = []
f.close()

print("Done :D\nCheck your folder for the results\n")

File: R/PrimeX.R
NL <- 1000000000

cat("========\n PrimeX\n========\n\n")

file_connection <- file("primes.txt", "w")

N <- NL + 2
primes <- rep(TRUE, N)

i <- 2
while (i < 3) {
    if (primes[i]) {
        write(i, file = file_connection, append = TRUE, sep = "\n")
    }

    j <- i
    while (j < N) {
        primes[j] <- FALSE
        j <- j + i
    }

    i <- i + 1
}

i <- 3
while (i < N) {
    if (primes[i]) {
        write(i, file = file_connection, append = TRUE, sep = "\n")
    }

    j <- i
    while (j < N) {
        primes[j] <- FALSE
        j <- j + i
    }

    i <- i + 2
}

primes <- NULL
close.connection(file_connection)

cat("Done :D\nCheck your folder for the results\n")